.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        scalaversion: ["2.11.12", "2.12.13", "2.13.5"]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v10
+        with:
+          java-version: "adopt@1.8"
+      - uses: coursier/cache-action@v5
+      - uses: actions/checkout@v2
+        with:
+          repository: sjrd/scala-js
+          ref: cc7232bc9f8135b968a4d405a58a454163a5940c
+          path: ./scalajs-nightly
+      - name: publishLocal Scala.js nightly
+        run: |
+          sbt ";ir2_12/publishLocal;linkerInterface2_12/publishLocal;linker2_12/publishLocal;testAdapter2_12/publishLocal;sbtPlugin/publishLocal"
+          sbt ";compiler2_11/publishLocal;library2_11/publishLocal;testInterface2_11/publishLocal;testBridge2_11/publishLocal;jUnitRuntime2_11/publishLocal;jUnitPlugin2_11/publishLocal"
+          sbt ";compiler2_12/publishLocal;library2_12/publishLocal;testInterface2_12/publishLocal;testBridge2_12/publishLocal;jUnitRuntime2_12/publishLocal;jUnitPlugin2_12/publishLocal"
+          sbt ";compiler2_13/publishLocal;library2_13/publishLocal;testInterface2_13/publishLocal;testBridge2_13/publishLocal;jUnitRuntime2_13/publishLocal;jUnitPlugin2_13/publishLocal"
+        working-directory: ./scalajs-nightly
```

Comment:
All this is only because we depend on snapshot, right?

Reply:
Yes, it's only because of the snapshot. Also once the other PR is merged, I'll adapt the `repository` and `ref` to point to `scala-js/scala-js`. And once 1.6.0 is out, we can remove all of this.